lib/Dialect/AIEVec/Transforms/ConvertVectorToAIEVec.cpp

```diff
@@ -89,6 +113,115 @@ struct UPDOpEffectiveAccessSizeAnalysis {
 //===----------------------------------------------------------------------===//
 // Lowering patterns
 //===----------------------------------------------------------------------===//
+// This pattern replaces `vector.fma` with `aievec.mac_elem`.
```

Comment:
1. This description is insufficient. 
2. The requirement that one of the operands of the vector.fma is a vector.broadcast is unnecessary.